# See Dynamixel documentation for details
float64 position
---
string result
